providers/openfeature-provider-flagd/src/openfeature/contrib/provider/flagd/resolvers/process/connector/grpc_watcher.py

```diff
@@ -0,0 +1,92 @@
+import json
+import logging
+import threading
+import time
+import typing
+
+import grpc
+
+from openfeature.event import ProviderEventDetails
+from openfeature.exception import ParseError
+from openfeature.provider.provider import AbstractProvider
+
+from ....config import Config
+from ....proto.flagd.sync.v1 import sync_pb2, sync_pb2_grpc
+from ..flags import Flag, FlagStore
+
+logger = logging.getLogger("openfeature.contrib")
+
+
+class GrpcWatcherFlagStore(FlagStore):
```

Comment:
Might be a good idea to split the store from the connector as the Java provider does, to better separate the responsibilities of each.